wip/services/exec/exec.go

```diff
@@ -13,7 +13,7 @@ var (
 )
 
 type Store interface {
-	Create(project string, exec types.Exec) error
+	Create(ctx context.Context, project string, exec types.Exec) error
```

Comment:
I've been using the convention of not passing in the context to the store. If we change this, we should change it for the entire interface.

Reply:
I think in the long term not passing context will mean that we can’t use the store for telemetry use cases. Most telemetry services operate using context and will collect information from database/sql using context. 

Let’s cross that bridge when we get to it because otherwise it’s premature optimisation